### DISCUSSION TITLE
[FEATURE-REQUEST] En el momento que el un usuario realiza un cambio en la tasks y le pone "Guardar", el sistema arroje un mensaje

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#247] [3. Ideas & Feature Requests] [answered]
Hago la idea a productos, para que sea como algo estándar. Puede que algún usuario dentro de una tasks cambie un parámetro y lo guarda, siendo que los demás usuarios no tengan conocimiento de qué parámetro cambio o quién cambió. Entonces creo que sería bueno incorporar ese mensaje desde la task y no de una implementación (debido a que s existen 10 implementaciones puede que el codigo o la forma sean diferente entre si.

Dejo un ss de la idea.
![image](https://user-images.githubusercontent.com/61419471/121223286-5e039300-c855-11eb-93d9-74fa6801c231.png)

Gracias :>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG Tienes razón, falta feedback al usuario de que la tarea fue editada exitosamente. Consideraremos agregar un snackbar parecido a lo que tu mencionas.
